code/modules/research/designs/medical_designs.dm

```diff
@@ -84,6 +84,39 @@
 	build_path = /obj/item/weapon/reagent_containers/glass/beaker/noreact
 	category = list("Medical Designs")
 
+/datum/design/bluespacesyringe
+	name = "Bluespace Syringe"
+	desc = "An advanced syringe that can hold 60 units of chemicals"
+	id = "bluespacesyringe"
+	req_tech = list("bluespace" = 2, "materials" = 2, "combat" = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_GLASS = 4000, MAT_PLASMA = 4000, MAT_DIAMOND = 2500)
+	reliability = 76
+	build_path = /obj/item/weapon/reagent_containers/syringe/bluespace
+	category = list("Medical Designs", "Bluespace Designs")
+
+/datum/design/noreactsyringe
+	name = "Cryo Syringe"
+	desc = "An advanced syringe that stops reagents inside from reacting. It can hold up to 20 units."
+	id = "noreactsyringe"
+	req_tech = list("materials" = 2, "combat" = 1)
+	build_type = PROTOLATHE
+	materials = list(MAT_GLASS = 4000, MAT_GOLD = 1000)
+	reliability = 76
+	build_path = /obj/item/weapon/reagent_containers/syringe/noreact
+	category = list("Medical Designs")
+
+/datum/design/piercesyringe
+	name = "Piercing Syringe"
+	desc = "A diamond-tipped syringe that pierces armor when launched at high velocity. It can hold up to 10 units."
+	id = "piercesyringe"
+	req_tech = list("materials" = 3, "combat" = 1)
+	build_type = PROTOLATHE
+	materials = list(MAT_GLASS = 4000, MAT_DIAMOND = 1500)
```

Comment:
This and the others are still too cheap, in my opinion. It should make you wince to waste one of these.